### PR TITLE
Format average, maximum, and distance units

### PR DESCRIPTION
**Pull request by andrewgaul**
Previously Speedmeter only applied imperial units to the current
speed.  Also isolate the presentation of units into the activity.

### CODE REVIEW

File: app/src/main/java/fly/speedmeter/grub/MainActivity.java [hunk 1]
```diff
@@ -74,13 +74,44 @@ protected void onCreate(Bundle savedInstanceState) {
...
+                if (sharedPreferences.getBoolean("miles_per_hour", false)) {
+                    maxSpeedTemp *= 0.62137119;
+                    distanceTemp = distanceTemp / 1000.0 * 0.62137119;
+                    averageTemp *= 0.62137119;
+                    speedUnits = "mi/h";
+                    distanceUnits = "mi";
+                } else {
+                    speedUnits = "km/h";
+                    if (distanceTemp <= 1000) {
+                        distanceTemp /= 1000.0;
```
Comment (flyingrub) [inline app/src/main/java/fly/speedmeter/grub/MainActivity.java]: Shouldn't we divide by 1000 when the distance is over 1000 not under ?

Comment (andrewgaul) [inline app/src/main/java/fly/speedmeter/grub/MainActivity.java]: Done.

Comment (flyingrub) [inline app/src/main/java/fly/speedmeter/grub/MainActivity.java]: the distance is in meter, it mean that you must divide by 1000 when you are in km.

Comment (andrewgaul) [inline app/src/main/java/fly/speedmeter/grub/MainActivity.java]: Done.

**Pull request merged**